### ISSUE TITLE
Error with uuid converter on generated files

### ISSUE THREAD

**Issue by max4t** [Raizlabs/DBFlow#1304]
DBFlow Version: 4.0.2
Issue Kind: Bug

Hi, 
when building my project, i have these two errors:
- `error: cannot find symbol variable global_typeConverterUUIDConverter` (on the generated `Refund_Table.java`)
- `error: incompatible types: String cannot be converted to UUID` (on the generated `Transfer_Table.java`)

Here is the sample of my tables (in kotlin)
```kotlin
@Table(database = AppDatabase::class)
@ManyToMany(
    generatedTableClassName = "Refund", referencedTable = Transfer::class,
    referencedTableColumnName = "refund_in", thisTableColumnName = "refund_out",
    saveForeignKeyModels = true
)
data class Transfer(
    @PrimaryKey var transfer_id: UUID = UUID.randomUUID(),
    @ForeignKey(stubbedRelationship = true) var origin: Account? = null,
    @ForeignKey(stubbedRelationship = true) var destination: Account? = null
)
```

So, looked into the generated files and `Refund_Table.java` never instantiate the uuid type converter like in the other tables:
```kotlin
public Refund_Table(DatabaseDefinition databaseDefinition) {
    super(databaseDefinition);
}
```

And the second error, Account id is a UUID but the string returned from the cursor is not converted with `global_typeConverterUUIDConverter.getModelValue`
```kotlin
@Override
public final void loadFromCursor(FlowCursor cursor, Transfer model) {
    ...
    model.getOrigin().setAccount_id(cursor.getString(index_origin_account_id_Account_Table));
    ...
}
```

Unless i'm wrong and i made a rookie mistake, these are bugs, right?

**Comment by agrosner** [comment]
I see, so the generated `@ManyToMany` class is not recognizing the type converter properly. makes sense.

**Comment by agrosner** [comment]
fixed in `develop`, will be in 4.0.3

**Comment by max4t** [comment]
ManyToMany table fixed indeed, thanks
